### DISCUSSION TITLE
Add nested bottom navigation bar to docs

### DISCUSSION THREAD

**Discussion by monhelle** [csells/go_router#254] [Q&A]
I love Go Router Mr. Sells. , it is totally awesome to use. It makes everything so much easier to read when you use context.goNamed. I also liked that you went on with Majid on Youtube to show how to implement Go router. Thank you for the detailed explanation on that show. I have also been looking at your docs.  I was wondering, is it possible to add nested bottom navigation bar to the gorouter.dev pages? I know that you have added nested navigation with tabs, but I think it would have been very useful to show it with the bottom navigation bar as well.

Best regards,
Monica

**Comment by techouse** [comment]
I think you're looking for this https://github.com/csells/go_router/issues/82#issuecomment-980614094
